test/foundry/GeometricPriceBook/Unit.t.sol

```diff
@@ -55,17 +55,19 @@ contract GeometricPriceBookUnitTest is Test {
             _testPriceToIndex(price, false, index);
             _testPriceToIndex(price, true, index);
             _testPriceToIndex(price + 1, false, index);
-            if (index == 0xffff) {
+            if (index == priceBook.maxIndex()) {
```

Comment:
It would be good to have tests for cases where the index exceeds the `maxIndex`.